--- a/themes/Sean/layout/chat/chat-tab.phtml
+++ b/themes/Sean/layout/chat/chat-tab.phtml
@@ -22,7 +22,7 @@
 							<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="24" height="24"><path fill="currentColor" class="hover_path" d="M6.455 19L2 22.5V4a1 1 0 0 1 1-1h18a1 1 0 0 1 1 1v14a1 1 0 0 1-1 1H6.455zM13 11h3l-4-4-4 4h3v4h2v-4z"></path><path fill="currentColor" d="M6.455 19L2 22.5V4a1 1 0 0 1 1-1h18a1 1 0 0 1 1 1v14a1 1 0 0 1-1 1H6.455zM4 18.385L5.763 17H20V5H4v13.385zM13 11v4h-2v-4H8l4-4 4 4h-3z"></path></svg> <?php echo $wo['lang']['open_in_new_tab'];?>
 						</a>
 					</li>
-					<?php if ($wo['config']['audio_chat'] == 1 && ($wo['config']['twilio_video_chat'] == 1 || $wo['config']['agora_chat_video'] == 1)) {
+					<?php if ($wo['config']['audio_chat'] == 1 && ($wo['config']['twilio_video_chat'] == 1 || $wo['config']['agora_chat_video'] == 1 || $wo['config']['p2p_chat_video'] == 1)) {
 						if ($wo['config']['can_use_audio_call']) {
 					?>
 					<?php if ($wo['chat']['recipient']['lastseen'] > (time() - 60)) { ?>
@@ -33,7 +33,7 @@
 						</li>
 					<?php } else { ?>
 					<?php } } } ?>
-					<?php if ($wo['config']['video_chat'] == 1 && ($wo['config']['twilio_video_chat'] == 1 || $wo['config']['agora_chat_video'] == 1)) {
+					<?php if ($wo['config']['video_chat'] == 1 && ($wo['config']['twilio_video_chat'] == 1 || $wo['config']['agora_chat_video'] == 1 || $wo['config']['p2p_chat_video'] == 1)) {
 						if ($wo['config']['can_use_video_call']) {
 					?>
 					<?php if ($wo['chat']['recipient']['lastseen'] > (time() - 60)) { ?>
